.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+<!--  Thanks for submitting a pull request! Here are some tips for you:
+1. Please make sure you have read and understood the contributing guidelines: https://github.com/alibaba/blob/master/CONTRIBUTING.md
```

Comment:
You may replace the URL with https://opensergo.io/docs/community/contribution-guidance/